config/locales/helpers/fr.yml

```diff
@@ -4,230 +4,229 @@ fr:
+        mod_roundrobin_notifications: Envoyez-moi des notifications occasionnelles sur le succès de la communauté
       users_setting:
-        feed_url: RSS Feed URL
+        feed_url: URL du flux RSS
     placeholder:
       organization:
         name: Acme Inc.
         slug: acmeinc
-        url: https://yoursite.com
-        location: Halifax, Nova Scotia
+        url: https://votresite.com
+        location: France, Toulouse
         email: mail@example.com
         company_size: "..."
         story: "..."
         tech_stack: "..."
         cta_body_markdown: "..."
-        cta_button_text: Visit Acme Site
-        cta_button_url: https://example.com
+        cta_button_text: Visiter le site Acme 
+        cta_button_url: https://votresite.com
         proof: "..."
       response_template:
         title: "..."
       user:
         name: John Doe
         email: john.doe@example.com
         username: johndoe
-        feed_url: https://yoursite.com/feed
+        feed_url: https://votresite.com/feed
     application_helper:
-      app_logo: App logo
+      app_logo: Logo de l'application
       follow:
         aria_label:
-          Organization: 'Follow organization: %{name}'
-          Tag: 'Follow tag: %{name}'
-          User: 'Follow user: %{name}'
-          default: 'Follow %{type}: %{name}'
+          Organization: "Suivre l'organisation: %{name}"
+          Tag: 'Suivre le tag: %{name}'
+          User: "Suivre L'utilisateur : %{name}"
+          default: "Suivre %{type}: %{name}"
         text:
           Organization: Suivre
           Tag: Suivre
           User: Suivre
           default: Suivre
-      required: Required
+      required: Requis
       subtitle:
-        week: Top posts this week
-        month: Top posts this month
-        year: Top posts this year
-        infinity: All posts
-        latest: Latest posts
+        week: Les meilleurs articles de cette semaine
+        month: Les meilleurs articles de ce mois
+        year: Les meilleurs articles de cette année
+        infinity: Tous les messages
+        latest: Derniers messages
       title_text: "%{title} - %{timeframe}"
     articles_helper:
       logo: "%{service_name} logo"
-      most_comments: Most Comments
-      most_reactions: Most Reactions
-      most_views: Most Views
-      recently_created: Recently Created
-      recently_published: Recently Published
-    authentication_helper:
-      any_of_those: any of those
-      email_password: Email & Password
-      reminder: 'Reminder: you used %{method} to authenticate your account, so please use %{dem} to sign in if prompted.'
-      that: that
-      invite_only: You cannot do this until you disable Invite Only Mode
+      most_comments: Le plus de commentaires
+      most_reactions: Le plus de réactions
+      most_views: Le plus de vues
+      recently_created: Récemment créé
+      récemment_publié: Récemment publié
```

Comment:
this key `récemment_publié` should be `recently_published` (to match the code calling the translation, this and the English key must be the same)

Reply:
Oowww good catch !